### PR TITLE
[PeerDAS] req / resp domain bug fixes

### PR DESCRIPTION
**Pull request by 0x00101010**
**What type of PR is this?**
Bug fix

**What does this PR do? Why is it needed?**
req / resp domain improvement & bug fixes

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: cmd/beacon-chain/flags/base.go [hunk 1]
```diff
@@ -174,6 +175,19 @@ var (
 		Usage: "The factor by which blob batch limit may increase on burst.",
 		Value: 2,
 	}
+	// DataColumnBatchLimit specifies the requested data column batch size.
+	DataColumnBatchLimit = &cli.IntFlag{
```
Comment (nalepae) [inline cmd/beacon-chain/flags/base.go:179]: There is some other places where `BlobBatchLimit` and `BlobBatchLimitBurstFactor` are used, and where the 2 new flags are not used.
With the current configuration, `batchSize` in https://github.com/0x00101010/prysm/blob/peerdas-rpc-req-resp/beacon-chain/sync/rpc_data_column_sidecars_by_root.go#L115-L116 is always 0.

Comment (nalepae) [inline cmd/beacon-chain/flags/base.go:179]: There is some other places where `BlobBatchLimit` and `BlobBatchLimitBurstFactor` are used, and where the 2 new flags are not used.
With the current configuration, `batchSize` in https://github.com/0x00101010/prysm/blob/peerdas-rpc-req-resp/beacon-chain/sync/rpc_data_column_sidecars_by_root.go#L115-L116 is always 0.

Comment (0x00101010) [reply cmd/beacon-chain/flags/base.go:179]: Can you elaborate a little bit more?

I understand that BlobBatchLimit is used in some other places, but I fail to see why with current configuration batchSize will always be 0? It is initialized in flags/config.go?

Comment (0x00101010) [reply cmd/beacon-chain/flags/base.go:179]: extracted the change out to https://github.com/prysmaticlabs/prysm/pull/14122

**Pull request closed without merging**